CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ set(LIBDBI_DIR "" CACHE PATH "Path to libdbi install directory")
 set(MAVEN_QUIET False CACHE BOOL "Do not print mvn messages")
 set(GENOMICSDB_SPARK_PROFILE "" CACHE STRING "Profile to choose Apache Spark version") # used in Maven builds
 set(IPPROOT "" CACHE PATH "Path to IPP libraries - used when optimized zlib is used")
+set(GENERATE_PROTOBUF_FILES_IN_BUILD_DIR False CACHE BOOL "Generate all protobuf files in build directory")
```

Comment:
Why is this needed?

Reply:
Just a convenience variable - if set all the generated files are in the build directory, none in the source. This allows you to create a build directory once (say for profiling) and then debug/develop the source independentlyor in parts.